src/utils/general.ts

```diff
@@ -894,8 +896,9 @@ export function waitForPageToBeVisible() {
 export async function clearCache() {
   const cacheArray = await api.storage.list();
   let deleted = 0;
+  if (api.type === 'webextension') localStore.clear();
```

Comment:
As this is a utility function. Meaning it can be used anywhere any time. I do not feel safe to clear the localstorage here.

I feel like putting it here is safer, as it makes it clearer that it should be executed in the context of the popup. While not totally safe, as the settings are build in a way that they could be injected anywhere. But still better than in the utility

https://github.com/MALSync/MALSync/blob/master/src/_minimal/components/settings/settings-structure-etc.ts#L44

Reply:
Yeah, now I think you are right, will fix it